starter-repos/app-template/skaffold/render.yaml

```diff
@@ -35,6 +35,10 @@ skaffold-render:
     - git config --global user.email "${CI_PROJECT_NAME}-ci@${CI_SERVER_HOST}"
     - git config --global user.name "${CI_PROJECT_NAMESPACE}/${CI_PROJECT_NAME}"
 
+    - |
+      sed -i "s/image:.*/image: app/g" config/envs/prod-central/deployment.yaml
+      sed -i "s/image:.*/image: app/g" config/envs/prod-east/deployment.yaml
+      sed -i "s/image:.*/image: app/g" config/envs/staging-west/deployment.yaml
```

Comment:
The infinite pipeline might be more confusing/disruptive than the failed pipeline upon code changes (i.e. the original skaffold render pipeline). Let's keep the original config for now; we can update this once the `skaffold` feature is available.